backend/src/routes/v1/webAde.js

```diff
@@ -181,8 +160,9 @@ webAde.post('/configForm', [
   } = req.body;
 
   // Check if this is allowed, return the error message if not
-  const err = utils.checkWebAdePostPermissions(req.user.jwt, configForm);
-  if(err) {
+  const err = await appConfigComponent.getPermissionError(req.user.jwt, configForm);
+  if (err) {
+    log.debug('/configForm', `No permission to Post WebADE config: ${err}`);
```

Comment:
Just as an extension of our previous conversation, should we be doing log statements like this on the middleware route layers? If we must, I'd suggest the following syntax:

```
log.debug('GET /configForm', `No permission to Post WebADE config: ${err}`);
```

Reply:
Switched syntax, will give some thought as to logging on route layer.. there are log.error statements around in routes